File: Python/pset6/cash.py
from cs50 import get_float

i = -1.00
count = 0
while i < 0:
    i = get_float("Change owed: ")

f = int(i * 100)
if f >= 25:
    count += int(f / 25)
    f %= 25
if f >= 10:
    count += int(f / 10)
    f %= 10
if f >= 5:
    count += int(f / 5)
    f %= 5
if f >= 1:
    count += int(f)
print(count)

File: C/pset1/cash.c
#include<cs50.h>
#include<stdio.h>
#include<math.h>
int main(void)
{
    float dollars;
    int cents,count=0;
    do
    {
        dollars=get_float("Change owed:");  

    }while(dollars<=0.009);
    cents=round(dollars*100);
    while(cents!=0)
    {
        if(cents>=25)
        {
            count+=cents/25;
            cents=cents%25;
        }
        else if(cents>=10)
        {
            count+=cents/10;
            cents=cents%10;
        }
        else if(cents>=5)
        {
            count+=cents/5;
            cents=cents%5;
        }
        else 
        {
            count+=cents;
            cents=cents%1;
        }
    }
    printf("%i \n",count);
}
